model/resource/cloud_provider/gcp/project.yaml

```
@@ -0,0 +1,12 @@
+groups:
+  - id: gcp.project
+    prefix: gcp.project
+    type: resource
+    brief: >
+      Project related attributes.
+    attributes:
+      - id: gcp.project_id
```

this would result in `gcp.project.gcp.project_id` attribute (since you have a prefix and repeat it in the attribute name).

If you expect more attributes in the `gcp.project` namespace, you probably want to change the name to `gcp.project.id` . You can do it with 

```suggestion
      - id: id
```

or 

```suggestion
  - id: gcp.project
    prefix: gcp
    type: resource
    brief: >
      Project related attributes.
    attributes:
      - id: project.id
```

---

Done